docs/UserGuide.md

```diff
@@ -108,7 +108,7 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS d/DEADLINE [t/TAG]…​
 * `NAME` is case-insensitive. `John Doe` and `joHN dOE` are considered same clients, but name is stored in the same case as the input (so `John Doe` is stored as `John Doe` and `JOHN Doe` is stored as `JOHN Doe`).
 * `PHONE_NUMBER` should be **numeric** digits, may include “-” or spaces. Example: `555-1234` or `555 1234`.
 * `EMAIL`  Standard email format: “user@example.com”.
-* `PAYMENT STATUS` Acceptable values are `pending`, `partial`, `paid`, `late`. Case-insensitive.
+* `PAYMENT STATUS` Acceptable values are `pending`, `partial`, `paid`, `late`. Case-insensitive. These values are displayed as an icon with the colours orange, yellow, green and red respectively.
```

Comment:
By any chance, could it be a good idea to give some examples of what the colours may look like/the situations you may use each payment status in?